action.yml

```
@@ -117,3 +140,4 @@ outputs:
 runs:
   using: node16
   main: dist/index.js
+  post: dist/cleanup.js
```

To keep backwards compatibility, I think we could add a new boolean input, e.g., `cleanup-in-post` and add a `post-if` directive here:

```suggestion
  post: dist/cleanup.js
  post-if: inputs.cleanup-in-post
```